vowpalwabbit/gen_cs_example.h

```diff
@@ -137,14 +139,14 @@ void gen_cs_label(cb_to_cs& c, example& ec, COST_SENSITIVE::label& cs_ld, uint32
         ((c.known_cost->cost - wc.x) * (c.known_cost->cost - wc.x) - c.avg_loss_regressors);
     c.last_pred_reg = wc.x;
     c.last_correct_cost = c.known_cost->cost;
-    wc.x += (c.known_cost->cost - wc.x) / c.known_cost->probability;
+    wc.x += (c.known_cost->cost - wc.x) / (std::max)(c.known_cost->probability, clip_p);
```

Comment:
Is there a reason why you sometimes use safe_probability() and sometimes not?

Reply:
I've keep everything as is. Not editing the use of safe_probability()